### ISSUE TITLE
Sounds missing when jumping to min/max for "Loops" spinner.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-electromagnetic-lab#193]
For https://github.com/phetsims/sun/issues/886.

Reported in https://github.com/phetsims/sun/issues/886#issuecomment-2231358075

Until https://github.com/phetsims/sun/issues/886 is addressed, I'll need to use the same workaround as https://github.com/phetsims/gas-properties/issues/281#issuecomment-2207001340.

**Comment by pixelzoom** [comment]
@Nancy-Salpepi please test in main, close if OK. As with https://github.com/phetsims/gas-properties/issues/281#issuecomment-2207110945, note that you will hear Home/End sounds even if you're already at min/max, due to limitations of the workaround.

**Comment by Nancy-Salpepi** [comment]
Sounds good in main.
Closing.
